manage_cortex_prs.sh

```
@@ -0,0 +1,435 @@
+    created=$(echo "$pr_data" | jq -r '.createdAt')
+    mergeable=$(echo "$pr_data" | jq -r '.mergeable')
+    review_count=$(echo "$pr_data" | jq -r '.reviews | length')
+    
+    # Calculate age
+    created_ts=$(date -j -f "%Y-%m-%dT%H:%M:%SZ" "$created" +%s 2>/dev/null || echo 0)
+    now_ts=$(date +%s)
+    age_days=$(( (now_ts - created_ts) / 86400 ))
+    
+    # Skip drafts
+    if [ "$is_draft" = "true" ]; then
+        draft_prs+=($pr_num)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)